### PR TITLE
Correction of Task Pattern command line arguments

### PR DESCRIPTION
**Pull request by lukasrothenberger**

### CODE REVIEW

File: discopop_explorer/__main__.py [hunk 1]
```diff
@@ -23,10 +23,12 @@
     --fmap=<fmap>               File mapping [default: FileMapping.txt]
     --json=<json_out>           Json output
     --plugins=<plugs>           Plugins to execute
-    --cu-inst-res=<cuinstres>   CU instantiation result file. Task Pattern Detector is executed if this option is set.
+    --task-pattern              Enables the Task Pattern Detection.
```
Comment (aryamazaheri) [inline discopop_explorer/__main__.py:26]: don't use camel case plz.
"Enables the task parallelism pattern identification,"

**Review by aryamazaheri — CHANGES_REQUESTED** [review]

File: discopop_explorer/__main__.py (hunk 1 above)
Comment (lukasrothenberger) [reply discopop_explorer/__main__.py]: should be fixed now.

**Comment by aryamazaheri** [comment]
@lukasrothenberger some tests are not passing. Can you plz check why they fail and fix the PR?

**Review by aryamazaheri — APPROVED** [review]
OK to merge

**Pull request merged**